### ISSUE TITLE
[Bug] I cannot copy or edit the current URL

### ISSUE THREAD

**Issue by andreicristianpetcu** [mozilla-mobile/fenix#4790]
## Steps to reproduce
https://vimeo.com/354376336
Open a tab and click the URL bar.
### Expected behavior
Search field is opened with the URL filled in.
If I want to copy or edit the address I can do it.
### Actual behavior
The search field is enpty and I have no way to share it
### Device information

* Android device: 
![Screenshot_20190619-235816_Settings](https://user-images.githubusercontent.com/922368/63212055-0c31cc00-c108-11e9-9327-69df0e992961.png)

* Fenix version: latest nightly

**Comment by pwd-github** [comment]
I literally just came to report this issue.

**Comment by mikeGolf08** [comment]
Same here -

    Android device: Android 9 on OnePlus 3T (A3003) Oxygen OS 9.0.5
    Fenix version: Nightly 190817 06:05 (Build #12290612)
    package: 8.0.0, 78354bc12
    lizard: 69.0-20190812090043

**Comment by andreicristianpetcu** [comment]
I found an app that I can share to in order to copy to clipboard. This is an ugly workaround.

**Comment by PopeRigby** [comment]
Same

```
Android version: 8.1.0
Fenix version: Nightly 190818 06:04 (Build #12300612)
Package version: 8.0.0, 78354bc12
Gecko version: 69.0-20190812090043
```

**Comment by ekager** [comment]
Thank you all for your reports. Will be fixed by #4800

**Comment by andreicristianpetcu** [comment]
@ekager works in latest nightly. Thank you!

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on RC 1.3.0-rc.3 with Huawei P9 Lite (Android 6), and Xiaomi Mi 8 Lite (Android 9).
